src/openrct2/actions/StaffSetColourAction.hpp

```diff
@@ -44,12 +44,14 @@ DEFINE_GAME_ACTION(StaffSetColourAction, GAME_COMMAND_SET_STAFF_COLOUR, GameActi
     void Serialise(DataSerialiser & stream) override
     {
         GameAction::Serialise(stream);
+
```

Comment:
Can you revert this change? It's not related to the PR (I think it's a leftover from a previous one you did)
```suggestion
```